docs/dataloaders.md

```diff
@@ -1,25 +1,60 @@
 ### The 1 paragraph explaination of DataLoader
 
 [DataLoader](https://github.com/facebook/dataloader) is a per-request cache, we use a new DataLoader on each API route.
-A DataLoader takes all of our external API requests and batches them together. This means that we should never end up 
+A DataLoader takes all of our external API requests and batches them together. This means that we should never end up
 calling the same request twice with complex queries.
 
+For example, with this query:
+
+```graphql
+{
+  # GET artists/popular
+  popular_artists(size: 5) {
+    artists {
+      # GET artist/:id
+      name
+      # GET artist/:id/artworks
+      artworks_connection(first: 5) {
+        edges {
+          node {
+            title
+            artist {
+              # GET artist/:id
+              name
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+See how in the above query we ask for `# GET artist/:id` twice, in the popular artists' name, and then later in the
+artwork's artist - using the dataloader pattern that call would only happen once. This also means that multiple API
+calls for the same artist when looping through artworks be coalesced into the request.
```

Comment:
Caching and batching aren’t related to each other, so the “this also means” part of the last sentence isn’t really necessarily true and could confuse a newcomer to dataloader. Batching probably needs its own entry.